Data/Functor/Foldable.hs

```diff
@@ -537,6 +537,11 @@ toFix = refix
 fromFix :: Corecursive t => Fix (Base t) -> t
 fromFix = refix
 
+hoistFix :: Functor f
```

Comment:
Btw, this is the type of a class member discussed in https://github.com/ekmett/adjunctions/pull/49

Reply:
Thanks, I'll try to remember to convert those into instances once a version of adjunctions which includes Functor1 gets released.

Reply:
Note: that `Functor1` in adjunctions doesn't include `Functor` constraint though. And @ekmett was thinking of different place for `Functor1` (name up to bikeshedding) to end up.